app/templates/search/briefs.html

```diff
@@ -23,12 +23,32 @@
 
 {% block content_summary %}
 <div class="intro">
-  <h1>Latest opportunities</h1>
+
+  <h1> Opportunities</h1>
+
+
+    <article class="content-main" style="float: right;text-align: right;margin-top: -2.5em;">
+Show:
+        {%  if link_args['status'] != 'live' %}
```

Comment:
you could probably do all this with 1 `if`, and is the indenting incorrect?

Reply:
Fixed indent and unified if block